n3fit/src/n3fit/model_trainer.py

```diff
@@ -32,6 +33,8 @@
 # Each how many epochs do we increase the integrability Lagrange Multiplier
 PUSH_INTEGRABILITY_EACH = 100
 
+# Create a tuple for the input information
+InputInfo = namedtuple("InputInfo", ["input_l", "split_l", "input_idx"])
```

Comment:
I find the description of these fields rather unclear. I end up having to read the code to understand what it does.